library/src/test/java/com/chuckerteam/chucker/api/ChuckerInterceptorTest.kt

```diff
@@ -496,5 +498,48 @@ internal class ChuckerInterceptorTest {
         assertThat(transaction.requestPayloadSize).isEqualTo(request.body!!.contentLength())
     }
 
+    @ParameterizedTest
+    @EnumSource(value = ClientFactory::class)
+    fun oneShotRequestBody_isNotAvailableToChucker(factory: ClientFactory) {
+        server.enqueue(MockResponse())
+        val client = factory.create(chuckerInterceptor)
+
+        val oneShotRequest = object : RequestBody() {
+            private val content = Buffer().writeUtf8("Hello, world!")
+            override fun isOneShot() = true
```

Comment:
Could you add a test also for duplex request here?

Reply:
I mentioned it in "how to test" section. It's not really feasible at HTTP level. The best I could do is to write tests for `ResponseProcessor`.

Reply:
My bad, I overlooked that 👍 Closing this comment.